### PR TITLE
CFWEB-85 Use of NGRX Standalone API for application config and bootstrapping

### PR DESCRIPTION
**Pull request by ssaring**
* NGRX v15 provides a new Standalone API for configuration of NGRX during application bootstrapping
  * Module-based API is not needed anymore
* Example app is now using this standalone API, also in the 'catalog' package which stays module-based for example purposes

### CODE REVIEW

**Review by MariusAquarius — APPROVED** [review]

**Comment by manuel-mauky** [comment]
Can you mix old module-based and new standalone config for NGRX? 
I'm just wandering why the catalogue feature was also refactored even though it is still using the module-approach for the rest of the feature.

I liked the idea to showcase both approaches and how to combine them in one app for the components itself and so I thought it would also be good to show this for ngrx.
Or are there technical reasons in this case that prevent combination of both approaches?

**Comment by ssaring** [comment]
We're already discussed this with S.Bley and agreed that the example app should demonstrate both ways, standalone as default and module-based for migration or complexity purposes. Also documented in the WIKI. Therefore the app root, bootstrap etc. is done standalone-based, only the catalog package is module-based.

Unfortunately once NGRX has been initialized the Standalone way, the old module-based approach can't be used in the module-based catalog package anymore. This was also the reason for the dependency problem I've contacted you before. 
In summary: it's still possible to use NGRX in the module-based packages as before, but the NGRX configuration for that package has to be done the standalone-way. That's not a that big issue IMHO.
